SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj

```diff
@@ -176,7 +181,8 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos";
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = SWIFT_PACKAGE;
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = Default;
+				SWIFT_VERSION = 4.0;
```

Comment:
Ahh, I see. It needs to be set to 3.2 to be able to build in Xcode8 since Xcode 8 don't have support for 4.0.